src/spelling_wordlist.txt

```diff
@@ -0,0 +1,249 @@
+GScan
+acknowledgement
+aldergrove
+analyse
+aren
+arg
+args
+auth
+baz
+behaviour
+boolean
```

Comment:
Strange that this isn't recognised. Is it because it's technically a proper noun so ought to be capitalised?

Reply:
OED says capitalized...